components/ias-adapter/tests/ias-adapter/Makefile

```diff
@@ -0,0 +1,34 @@
+APP_NAME = ias-adapter-tests
+COMPONENT = ias-adapter
+APP_PATH = tests/ias-adapter
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.20
```

Comment:
Is there a specific reason for the e2e tests to be in the components themselves instead of in the "root" tests directory where other e2e tests are? Also, I didn't find `TestDefinition` CRD(from octopus) similar to the other components, so I'm wondering how the e2e tests will be executed?
Currently, we create a cluster test suite(also a CRD from octopus) that manages the mentioned test definitions and execute them in the cluster. For part of the tests we need external-services-mock and we temporarily deploy it on the cluster during the e2e tests execution.
Will the new component's e2e tests work "outside" this approach that we have